Tests/conf.json

```diff
@@ -610,11 +610,6 @@
             "integrations": "Cisco Umbrella Investigate",
             "playbookID": "Cisco Umbrella Test"
         },
-        {
```

Comment:
Dont remove it, just add to skip

Reply:
Ah okay, thought it had to be removed to skip. Will fix now